sql/src/test/java/io/crate/operation/aggregation/impl/PercentileAggregationTest.java

```diff
@@ -0,0 +1,186 @@
+/*
+ * Licensed to CRATE Technology GmbH ("Crate") under one or more contributor
+ * license agreements.  See the NOTICE file distributed with this work for
+ * additional information regarding copyright ownership.  Crate licenses
+ * this file to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.  You may
+ * obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.  See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * However, if you have executed another commercial license agreement
+ * with Crate these terms will supersede the license and you may use the
+ * software solely pursuant to the terms of the relevant commercial agreement.
+ */
+
+package io.crate.operation.aggregation.impl;
+
+import com.google.common.collect.ImmutableList;
+import io.crate.Streamer;
+import io.crate.analyze.symbol.Aggregation;
+import io.crate.core.collections.ArrayBucket;
+import io.crate.core.collections.Row;
+import io.crate.metadata.FunctionIdent;
+import io.crate.operation.aggregation.AggregationFunction;
+import io.crate.operation.aggregation.AggregationTest;
+import io.crate.operation.collect.InputCollectExpression;
+import io.crate.types.DataType;
+import io.crate.types.DataTypes;
+import org.elasticsearch.common.collect.HppcMaps;
+import org.elasticsearch.common.io.stream.BytesStreamOutput;
+import org.elasticsearch.common.io.stream.StreamInput;
+import org.elasticsearch.search.aggregations.metrics.percentiles.tdigest.TDigestState;
+import org.junit.Test;
+
+public class PercentileAggregationTest extends AggregationTest {
+
+    private Object[][] executeAggregation(DataType dataType, Object[][] data) throws Exception {
```

Comment:
Unless I'm misunderstanding the implementation, the executeAggregation function in the base class only accepts one input argument whereas percentile has 2 - the percentile and the column?